src/main/java/frc/team5115/Robot/RobotContainer.java

```diff
@@ -49,6 +51,7 @@ public class RobotContainer {
     private final Intake intake;
     private final Shooter shooter;
     private final DigitalInput reflectiveSensor;
+    private AutoAimAndRange a;
```

Comment:
I know it's tempting to use single letter variable names, especially for these longer class names, but it'll be harder to maintain in the long run. 

I always picture myself as the poor fool who has to read my own code in 6 months, and try to be as obvious and clear as possible. I've never regretted the extra time to write more characters.

Reply:
Alright, I'll try changing it to be more specific